.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: CI robot-browserstack sample
+on:
+  push:
```

Comment:
I guess trigger on pull_request would suffice, otherwise merging commit would also trigger tests.

Reply:
added that in case someone directly merges into the SDK branch..

Reply:
will update